src/components/Match/TeamfightMap/index.jsx

```diff
@@ -267,7 +267,7 @@ export const Tombstones = ({
 }) => (
   <div>
     {deathPositions.map((position, index) => (
-      <div key={index}>
+      <div>
```

Comment:
a key is required to get re-renders to work I believe

Reply:
I couldn't see any differences when testing but I'll revert it. Leaving it in currently violates `react/no-array-index-key`. Perhaps we will need ESLint to ignore this?